docs/variable.properties

```diff
@@ -1,2 +1,2 @@
 current_version=0.2.6
-worker_sdk_version=13.8
\ No newline at end of file
+worker_sdk_version=14.0
```

Comment:
14.0.1?

Reply:
there's no doc versions for patch releases

Reply:
Wot he said ^